app-utils/dmenu/autobuild/defines

```diff
@@ -5,3 +5,4 @@ PKGDES="A generic menu for X"
 
 ABTYPE=plainmake
 AB_FLAGS_SPECS=0
+ABSPLITDBG=0
```

Comment:
Why?

Reply:
Well, then it means that the flags weren't passed in properly, unless it's an ELF-less package.